R/derive_var_trtemfl.R

```diff
@@ -162,46 +235,79 @@ derive_var_trtemfl <- function(dataset,
                                end_window = NULL,
                                ignore_time_for_trt_end = TRUE,
                                initial_intensity = NULL,
-                               intensity = NULL) {
+                               intensity = NULL,
+                               group_var = NULL,
+                               subject_keys = get_admiral_option("subject_keys")[2]) {
```

Comment:
```suggestion
                               subject_keys = get_admiral_option("subject_keys")) {
```
We don't have any rules or conventions on the number of elements in "subject_keys" or which variable is placed where. You could have `exprs(USUBJID, STUDYID)`, `exprs(STUDYID, SITEID, SUBJID)`, or `exprs(USUBJID)`. I.e., selecting a certain element is not a good idea because in general you don't know what you get.